Assets/Scripts/Audio/AudioGroup.cs

```diff
@@ -4,46 +4,64 @@
  */
 
 [System.Serializable]
-public class AudioGroup : AudioData {
+public class AudioGroup : AudioData
+{
 	RandomizedQueue<AudioFile> fileQueue;
 	AudioFile currentFile;
 
-	public AudioGroup (params AudioFile[] files) {
+	public AudioGroup(params AudioFile[] files) 
+	{
 		checkFileQueue();
-		foreach (AudioFile file in files) {
+		foreach(AudioFile file in files) 
+		{
 			fileQueue.Enqueue(file);
 		}
 	}
 
-	public void AddFile (AudioFile file) {
+	public void AddFile(AudioFile file) 
+	{
 		checkFileQueue();
 		fileQueue.Enqueue(file);
 	}
 
-	void checkFileQueue () {
-		if (fileQueue == null) {
+	void checkFileQueue()
+	{
+		if(fileQueue == null) 
+		{
 			fileQueue = new RandomizedQueue<AudioFile>();
 		}
 	}
 
-	public AudioFile GetRandomFile () {
+	public AudioFile GetRandomFile() 
+	{
 		checkFileQueue();
 		return (currentFile = fileQueue.Cycle());
```

Comment:
I don't think you need the outer parentheses here, but you don't have to change it.